apps/sms/js/thread_ui.js

```diff
@@ -2735,9 +2735,9 @@ var ThreadUI = {
       tel = number || '';
     }
 
-    Contacts.findByAddress(number, function(results) {
-      var isContact = results && results.length;
-      var contact = results[0];
+    Contacts.findByAddress(number).then(function(contacts) {
```

Comment:
Okay, both ways work for me. I just noticed that you changed some functions to arrow functions that is why I was wondering how that one was different :)